README.md

```diff
@@ -1 +1,56 @@
-# portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My Portfolio Website
+
+> A portfolio website for my personal projects
+
+
+## Built With
+
+- Major languages: HTML, CSS and Javascript
+- Frameworks: Nodejs
+- Technologies: Github and Vscode
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+ - Clone the repository into your local machine
+ - Open the cloned file in Vscode
+ - Host locally  
+
+### Prerequisites
+  - An understanding of HTML, CSS and Javascript
+  - A github account and Vscode installed in local machine
+
+### Setup
+  - Linters configurations and starter files
+### Install
+  - Vscode, Node and NPM
+### Usage
+
+### Run tests
+  - Github actions and local linters test
+### Deployment
+
+
+
+## Authors
+
+👤 **Joshua Eze**
+
+- GitHub: [@Allenkeys](https://github.com/Allenkeys)
+- Twitter: [@jdgraay](https://twitter.com/jdgraay)
+- LinkedIn: [Eze Joshua](https://linkedin.com/in/eze-joshua)
+
+
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
_[Optional]_ I suggest that you remove this note from the README file. It's only a note to you (the author) not to your audience. By doing so, your documentation will be more professional and concise.